File: Session_One/Oct_14/beginner/greet.cpp
#include <iostream>

int main() {

  std::string name;
  std::string pokemon;
  std::cout << "Hello! What's your name?\n";
  std::cin >> name;
  std::cout << "Nice to meet you, " << name << "\n";
  std::cout << "What is your favorite Pokemon?\n";
  std::cin >> pokemon;
  std::cout << name<< "\'s favorite Pokemon is " << pokemon;
  return 0;
}

File: Session_One/Oct_14/beginner/Greet.java
import java.util.Scanner;

class Greet {
    public static void main(String[] args) {
       Scanner writer = new Scanner(System.in);  
       System.out.println("Hello! what is your name?"); 
       String name = writer.nextLine(); 
       System.out.println("nice to meet you " + name + "."); 
       System.out.println("What is your favorite Pokemon?"); 
       String pokemon = writer.nextLine(); 
       System.out.println(name + "\'s favorite Pokemon is " + pokemon + "."); 
    }
}
